### PR TITLE
Update info for PyPI release

### PR DESCRIPTION
**Pull request by krisgesling**
Adds README as long description for PyPI listing, as well as some general metadata.

Also increments version number for new release.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -35,9 +38,21 @@
     maintainer='Matthew Scholefield',
     maintainer_email='matthew331199@gmail.com',
     description='Mycroft Skills Kit',
+    long_description=long_description,
+    long_description_content_type="text/markdown",
     entry_points={
         'console_scripts': {
             'msk=msk.__main__:main'
         }
     }
```
Comment (forslund) [inline setup.py]: Missing comma

File: setup.py [hunk 2]
```diff
@@ -21,9 +21,12 @@
 # under the License.
 from setuptools import setup
 
+with open("readme.md", "r") as fh:
```
Comment (forslund) [inline setup.py]: Should be "README.md"

**Review by forslund — CHANGES_REQUESTED** [review]
There were a couple of minor issues I needed to change to get the setup.py running. See comments in code.

**Comment by krisgesling** [comment]
Good catch, I clearly didn't test with a run of setuptools :blush:

**Review by forslund — APPROVED** [review]
Works now!

**Pull request merged**